configs/platforms/windows-2012r2-x64.rb

```diff
@@ -21,5 +21,5 @@
 
   plat.platform_triple "x86_64-unknown-mingw32"
 
-  plat.package_type "nuget"
+  plat.package_type "msi"
```

Comment:
I would say lets leave it in until we are farther in to the process, for now I think it's good to document we are using very new functionality.